<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginSet</class>
 <widget class="QMainWindow" name="LoginSet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget01_2" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>5</number>
         </property>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>密码:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>用户名:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,0">
           <property name="spacing">
            <number>6</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="creUserBtn">
             <property name="text">
              <string>立即创建</string>
             </property>
             <property name="type" stdset="0">
              <string>primary</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="1" column="2">
          <widget class="QLineEdit" name="userPass">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <layout class="QHBoxLayout" name="horizontalLayout_8"/>
         </item>
         <item row="0" column="2">
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLineEdit" name="userName">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>40</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>300</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="leftMargin">
          <number>5</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>功能区:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="0,0,0,0,0,0,0">
           <property name="spacing">
            <number>6</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLineEdit" name="findUserName">
             <property name="minimumSize">
              <size>
               <width>300</width>
               <height>40</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="findBtn">
             <property name="text">
              <string>查询</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>-</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="delBtn">
             <property name="text">
              <string>删除</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>-</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="fixBtn">
             <property name="text">
              <string>修改</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="1" column="1">
          <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0">
           <item>
            <widget class="QTableWidget" name="tableWidget">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <column>
              <property name="text">
               <string>用户名</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>密码</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>创建时间</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
